frame/im-online/src/lib.rs

```diff
@@ -430,7 +429,6 @@ pub mod pallet {
 	/// For each session index, we keep a mapping of `ValidatorId<T>` to the
 	/// number of blocks authored by the given authority.
 	#[pallet::storage]
-	#[pallet::getter(fn authored_blocks)]
```

Comment:
supposed to be removed? (there's not a supplementary verbosely defined getter defined for this one)

Reply:
Thanks, I'll fix it